### PR TITLE
238: fixing lint issues with imports

### LINKED ISSUE #238: Resolve non-invasive lint issues
Issues with imports:
- named import
- unused imports
- global imports

### PR DESCRIPTION
**Pull request by jac18281828**
closes #238 

This cleans up many minor issues with imports.     All remaining lint warnings are invasive issues that should be considered on their own.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "test": "forge test -v",
     "prettier:check": "prettier --check \"(contracts|test|script)/**/*.sol\"",
     "prettier:fix": "prettier --write \"(contracts|test|script)/**/*.sol\"",
-    "lint": "solhint 'contracts/**/*.sol' 'test/**/*.sol' 'script/**/*.sol'",
+    "lint": "solhint 'contracts/**/*.sol' 'script/**/*.sol'",
```
Comment (blockgroot) [inline package.json:10]:
```suggestion
    "lint": "solhint -w 3 'contracts/**/*.sol' 'script/**/*.sol'",
```

File: contracts/PermissionlessNodeRegistry.sol
Comment (blockgroot) [inline contracts/PermissionlessNodeRegistry.sol:1]: // solhint-disable no-inline-assembly at https://github.com/stader-labs/ethx/pull/239/files?diff=split&w=0#diff-01f44aaaf764a9d38dc6091fb156bb0a587f44d6108d33631a0855a68a8eb06cL558

File: .solhint.json [hunk 2]
```diff
@@ -3,6 +3,14 @@
...
+    "avoid-sha3": "warn",
+    "var-name-mixedcase": "off",
+    "func-name-mixedcase": "off",
+    "func-visibility": [
+      "warn",
+      {
+        "ignoreConstructors": true
+      }
+    ]
   }
```
[the remark this answers is not in the document]
Comment (jac18281828) [reply .solhint.json:15]: I prefer to review each of these instances individually.   We should keep our state size small if possible

File: package.json (hunk 1 above)
Comment (jac18281828) [reply package.json:10]: Unfortunately we are not ready for this change yet.   Hopefully very soon!  And we have made strides!

**Review by blockgroot — APPROVED** [review]

**Pull request merged**